sass/layout/_sidebars.scss

```diff
@@ -7,6 +7,25 @@
 	padding: $size__default-padding;
 }
 
+.sidebar-nav {
+	margin-right: 0;
+}
+
+.sidebar-nav .current_page_item > a {
+	color: $color__link;
+	font-weight: 400;
+}
+
+.sidebar-nav li {
+	list-style: none;
+	text-indent: -0.8em;
```

Comment:
This doesn't seem right. Negative text indent is usually only used to hide things. I think maybe you're trying to hide the bullets that show on `ul` elements? We should use `list-style-type: none;` instead.